code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -46,6 +47,9 @@
 		M.visible_message("<span class='danger'>[user] forces [M] to [apply_method] [src].</span>", \
 							"<span class='userdanger'>[user] forces [M] to [apply_method] [src].</span>")
 
+	makes_me_think = pick(strings("redpill.json", "redpill_questions"))
+	if(icon_state == "pill4" && prob(1)) //you take the red pill - you stay in Wonderland, and I show you how deep the rabbit hole goes

```

Comment:
why not make makes_me_think a var here instead of on the pill itself?

Reply:
Do you mean like this?
`var/makes_me_think = pick(strings("redpill.json", "redpill_questions"))`

Reply:
yup